src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -4,10 +4,8 @@
 import java.util.Date;
 import java.util.List;
 import java.util.Set;
-import org.broadinstitute.consent.http.db.mapper.DataAccessRequestDataMapper;
 import org.broadinstitute.consent.http.db.mapper.DataAccessRequestMapper;
 import org.broadinstitute.consent.http.db.mapper.DataAccessRequestReducer;
-import org.broadinstitute.consent.http.enumeration.EmailType;
```

Comment:
Thank you!  I'm looking forward to spotless.

Reply:
Me too 👍🏽